setup.py

```diff
@@ -137,7 +137,8 @@ def run(self):
         'traittypes',
         'traitlets',
         'Pillow',
-        'ipywebrtc'
+        'ipywebrtc',
+        'requests'
```

Comment:
I would point out that the python documentation directly recommends the request package: https://docs.python.org/3/library/urllib.request.html. But I can change to `urllib.request.urlretrieve` if you want?